packages/lavapack/test/runtimeEcma.js

```diff
@@ -0,0 +1,18 @@
+const ECMA_VERSION = 2019;
```

Comment:
consider making this a sys.env var?

Reply:
@naugtur my 5c on the question "do we still want to keep this but change it to 2020?":

Having a compliance test like introduced here is great. Though I'd also like to put this elsewhere. How about adding a non-standard field in `package.json` for it?

Reply:
I explicitly meant for this to be defined in the test - the way tests usually are redundant so that they specify the same assumptions, but independently. That's the value of having tests and that's why I don't want to read the number from eslint config or elsewhere. This test must fail when/if we go above es2020

Reply:
The in the current version does make it simpler  :+1: